### ISSUE TITLE
Can't get the hotkey to work

### ISSUE THREAD

**Issue by dre3002** [nbusseneau/BetterCartographyTable#6]
I install the mod but when I press the default hotkey "LeftShift", nothing happens. I can't change the pins to public/private and the colors does not change. I tried changing the hotkey but same result. I am unsure if maybe this is a compatibility issue so here is the list of mods I am currently using.

[Page1](https://prnt.sc/k4OYVycBL2_j)

[Page2](https://prnt.sc/InFg2N7dyO0c)

I don't think any of these mods would conflict but I am unsure.

**Comment by nbusseneau** [comment]
Asocial cartography should definitely conflict here, I think you must choose one or the other considering the overlap 😅

**Comment by dre3002** [comment]
> Asocial cartography should definitely conflict here, I think you must choose one or the other considering the overlap 😅

Ahh sorry that was my bad. I had tried your mod before installing that one. The screenshot was taken after I had installed AC which I forgot about lol. I am currently using Asocial cartography because that one works for me. It's just not as customizable as yours.


So just to be clear, I tried your mod without asocial cartography installed, but all of the other mods on the list were installed.
